src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -763,14 +771,21 @@ private interface BuiltInProfile {
      */
     ICC_Profile(Profile p) {
         cmmProfile = p;
+        builtIn = false;
     }
 
     /**
      * Constructs an {@code ICC_Profile} object whose loading will be deferred.
      * The ID will be 0 until the profile is loaded.
+     *
+     * <p>
+     * Note: {@code ProfileDeferralInfo} is used for only built-in profile
+     * creation and all built-in profiles should be constructed using it.
+     * </p>
```

Comment:
</p> is not necessary here.

Reply:
This note was added based on the suggestion here: https://github.com/openjdk/jdk/pull/23606#discussion_r1976123437

Reply:
Oh you meant `<p>` tag :) 
It was not rendered in github comment.